packages/dashboards/addon/components/dashboard-header.hbs

```diff
@@ -0,0 +1,127 @@
+{{!-- Copyright 2021, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
+<div class="container-full m-b-4 navi-dashboard__breadcrumb">
+  <div class="row is-marginless p-t-10">
+    <div class="xs-col-1-1 sm-col-1-2 col-1-5">
```

Comment:
The breadcrumb doesn't is the only column in the row correct? Isn't it always `1-1`?

Reply:
If I dont specify the column configuration for xs explicitly, it applies the default col-1-5